### PR TITLE
Improve focus state of delete button

### PR DESCRIPTION
**Pull request by tintinthong**
**Before**

Screenshot 2023-12-19 at 10 26 30

**After**

Screenshot 2023-12-19 at 10 25 50

### CODE REVIEW

**Review by burieberry — COMMENTED** [review]
Better but still looks a bit awkward. Can't we use a Boxel::Button component (kind='secondary-light') with built-in hover/focus states? Or maybe make the top and bottom padding smaller

**Comment by jurgenwerk** [comment]
I would expect that text only buttons have a focus state that is also text only (maybe just a different color)

**Review by burieberry — COMMENTED** [review]
Button component update looks good. I made a small update to the icons to replace the trash icon with the one from the designs, and updated a couple other ones that were missing stroke and looked anemic when used. Extended the focus style to the module delete icon as well in `detail-panel` gts file.

**Comment by burieberry** [comment]
> I would expect that text only buttons have a focus state that is also text only (maybe just a different color)

It's good to have that focus circle as well for better accessibility

**Comment by tintinthong** [comment]
> Button component update looks good. I made a small update to the icons to replace the trash icon with the one from the designs, and updated a couple other ones that were missing stroke and looked anemic when used. Extended the focus style to the module delete icon as well in `detail-panel` gts file.

Nice thanks. I went thru the changes and they look good particularly the spacing for the button on :focus state

**Comment by tintinthong** [comment]
> > I would expect that text only buttons have a focus state that is also text only (maybe just a different color)
> 
> It's good to have that focus circle as well for better accessibility

I think I agree with @burieberry

@jurgenwerk initially I thought your suggestion was the best but after consideration of accessibilty, I think it is difficult to choose a tasteful color palette for focus on delete button that gives a CLEAR contrast - in particular for the text-only option

I tried to change focus:hover to a `var(boxel-dark)` text and icon color 

https://github.com/cardstack/boxel/assets/8165111/1a0bc211-8c37-422d-a737-8be8bd86ff7b

Even in the scenario of the "trash" icon. I don't think it is super clear

https://github.com/cardstack/boxel/assets/8165111/c9ac6aba-5944-4628-955e-d8bbe3852722

I think the below solution might still be the best 

https://github.com/cardstack/boxel/assets/8165111/3b4885f7-4984-4ce2-ab17-bb5594e7dbfc

What do you guys think?

**Comment by burieberry** [comment]
> What do you guys think?

Looks good to me!

**Review by burieberry — APPROVED** [review]

File: packages/boxel-ui/addon/raw-icons/copy.svg [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <svg xmlns="http://www.w3.org/2000/svg" width="14" height="14" viewBox="0 0 14 14">
     <g data-name="Group 13494" transform="translate(1 1)">
-        <rect data-name="Rectangle 5051" width="7.8" height="7.8" rx="1" transform="translate(4.2 4.2)" style="fill:none;stroke:#03c4bf;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/>
-        <path data-name="Path 85688" d="M3.8 9.8h-.6A1.2 1.2 0 0 1 2 8.6V3.2A1.2 1.2 0 0 1 3.2 2h5.4a1.2 1.2 0 0 1 1.2 1.2v.6" transform="translate(-2 -2)" style="fill:none;stroke:#03c4bf;stroke-linecap:round;stroke-linejoin:round;stroke-width:2px"/>
+        <rect data-name="Rectangle 5051" width="7.8" height="7.8" rx="1" transform="translate(4.2 4.2)"
+            style="fill:none;stroke:var(--icon-color, #000);stroke-linecap:round;stroke-linejoin:round;stroke-width:var(--icon-stroke-width, 1.5px)" />
```
Comment (tintinthong) [inline packages/boxel-ui/addon/raw-icons/copy.svg:4]: @burieberry I changed this stroke width to be compatible with the trash icon that u changed. The stroke line is thinner

**Pull request merged**